README.md

```diff
@@ -117,9 +116,9 @@ A big feature of ctl is the support for multiple logs at a time. As all of ctl u
 Additionally, this feature supports the `-f, --follow` option which constantly streams logs.
 
 ## Run
-The run command is a wrapper over `kubectl run`. It requires cluster config to set up.
+The run command is a wrapper over `kubectl apply`. It requires cluster config to set up.
 
-The syntax is `ctl run NAME [flags]`. The label flags can be used to narrow down which cluster to run on. If there are multiple clusters in which a run is specified on, ctl randomly picks one.
+The syntax is `ctl run APPNAME [flags]`. The label flags can be used to narrow down which cluster to run on. If there are multiple clusters in which a run is specified on, ctl randomly picks one.
```

Comment:
didn't we change `run` to `up`

Reply:
Still need to update the readme, was going to do it after everything is approved